src/app/datagrid/selection/selection.html

```diff
@@ -145,7 +157,11 @@ <h3>Server-driven, trackBy item</h3>
     <ng-container *clrDgHideableColumn> Favorite color </ng-container>
   </clr-dg-column>
 
-  <clr-dg-row *ngFor="let user of serverTrackByIdUsers; trackBy: trackById" [clrDgItem]="user">
+  <clr-dg-row
+    *ngFor="let user of serverTrackByIdUsers; trackBy: trackById"
+    [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.